### PR TITLE
Founders page

### PR DESCRIPTION
**Pull request by mikemaddem**
linting and builds should pass now - had a bad npm install on my desktop

### CODE REVIEW

**Comment by mikemaddem** [comment]
@RITesports/director @RITesports/captains

**Comment by evanextreme** [comment]
@Rag4214 are the hound style violations something we should care about? Otherwise this looks good

**Comment by mikemaddem** [comment]
@evanextreme they're resolved so yeah nothing to worry about.

File: react-app/src/components/header/header-view.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ const Header: React.FC = () => (
       <Navbar.Collapse id="responsive-navbar">
         <Nav className="ml-auto text-center">
           <Nav.Link as={Link} to="/" eventKey="home" className="mr-md-5">Home</Nav.Link>
+          <Nav.Link as={Link} to="/founders" eventKey="founders" className="mr-md-5">Founders</Nav.Link>
```
Comment (evanextreme) [inline react-app/src/components/header/header-view.tsx]: @rag4214 do we want the founders link before teams / store? i'm thinking its more of a footer thing

**Comment by mikemaddem** [comment]
waiting for a proper image for the page

**Comment by evanextreme** [comment]
@mikemaddem what image are we waiting for

**Pull request closed without merging**